src/main/java/com/ft/methodearticlemapper/resources/PostContentToTransformResource.java

```diff
@@ -88,7 +90,9 @@ private void validateUuid(String uuid, EomFile eomFile) {
             throw ClientError.status(400).context(null).reason(ErrorMessage.UUID_REQUIRED).exception();
         }
         try {
-
+            if (!uuidPattern.matcher(uuid).matches()) {
+                throw new IllegalArgumentException();
```

Comment:
Looks like a nicer solution, will test this, and use it if it is really full proof.